test/trajectory_test.jl

```diff
@@ -0,0 +1,101 @@
+using DiffEqBase, DiffEqJump
```

Comment:
Is this test adding something over the other tests? If so, it should be added to runtests too, if not remove it.